src/functions/retryProcessing/framework/database/query-templates.ts

```diff
@@ -71,30 +71,54 @@ export const updateErrorsToAbortQueryTemplate = `
   SET tr.result_status = (SELECT id FROM RESULT_STATUS WHERE result_status_name = 'ERROR')
 `;
 
-export const updateManuallyIntervenedForReprocessQuery = `
-  UPDATE TEST_RESULT tr
-  JOIN UPLOAD_QUEUE uq
+export const manualInterventionReprocessUploadQueueQuery = `
+  UPDATE UPLOAD_QUEUE uq
+  JOIN TEST_RESULT tr
     ON tr.application_reference = uq.application_reference
     AND tr.staff_number = uq.staff_number
+    AND tr.result_status = (SELECT id FROM RESULT_STATUS WHERE result_status_name = 'PENDING')
+    AND uq.upload_status = (SELECT id FROM PROCESSING_STATUS WHERE processing_status_name = 'FAILED')
   SET
-    tr.result_status = (SELECT id FROM RESULT_STATUS WHERE result_status_name = 'PROCESSING'),
     uq.upload_status = (SELECT id FROM PROCESSING_STATUS WHERE processing_status_name = 'PROCESSING'),
     uq.retry_count = 0,
     uq.error_message = NULL
-  WHERE
-    tr.result_status = (SELECT id FROM RESULT_STATUS WHERE result_status_name = 'PENDING')
-    AND uq.upload_status = (SELECT id FROM PROCESSING_STATUS WHERE processing_status_name = 'FAILED')
+`;
+
+export const manualInterventionUploadQueueReplacementQuery = `
+  INSERT IGNORE INTO UPLOAD_QUEUE (
+    SELECT
+      tr.application_reference,
+      tr.staff_number,
+      NOW() as timestamp,
+      it.id as interface,
+      (SELECT id FROM PROCESSING_STATUS WHERE processing_status_name = 'PROCESSING') as upload_status,
+      0 as retry_count,
+      NULL as error_message
+  FROM TEST_RESULT tr, INTERFACE_TYPE it
+  WHERE tr.result_status = (SELECT id FROM RESULT_STATUS WHERE result_status_name = 'PENDING')
+  )
+`;
+
+export const manualInterventionReprocessTestResultQuery = `
+  UPDATE TEST_RESULT
+  SET result_status = (SELECT id FROM RESULT_STATUS WHERE result_status_name = 'PROCESSING')
+  WHERE result_status = (SELECT id FROM RESULT_STATUS WHERE result_status_name = 'PENDING')
 `;
 
 export const deleteAccepetedUploadsQuery = `
-  DELETE UPLOAD_QUEUE FROM UPLOAD_QUEUE
+  DELETE uq
+  FROM UPLOAD_QUEUE uq
+  JOIN TEST_RESULT tr
+    ON uq.application_reference = tr.application_reference
+    AND uq.staff_number = tr.staff_number
+    AND tr.result_status = (SELECT id FROM RESULT_STATUS WHERE result_status_name = 'PROCESSED')
   JOIN (
     SELECT application_reference, staff_number, interface
     FROM UPLOAD_QUEUE
     WHERE upload_status = (SELECT id FROM PROCESSING_STATUS WHERE processing_status_name = 'ACCEPTED')
     AND timestamp < ?
   ) to_delete
-    ON UPLOAD_QUEUE.application_reference = to_delete.application_reference
-    AND UPLOAD_QUEUE.staff_number = to_delete.staff_number
-    AND UPLOAD_QUEUE.interface = to_delete.interface
+    ON uq.application_reference = to_delete.application_reference
+    AND uq.staff_number = to_delete.staff_number
+    AND uq.interface = to_delete.interface;
```

Comment:
Is this semi-colon a typo?

Reply:
Good spot, it doesn't cause any errors, but it's unnecessary and none of the other queries have it, removed.